### ISSUE TITLE
Blacklist tags when parsing

### ISSUE THREAD

**Issue by kaytrance** [codelucas/newspaper#893]
Hi!

Is there a way to blacklist certain tags so that any text inside them will not be parsed and skipped entirely? For example when I parse a page I want to exclude everything inside `<code>` or `<pre>` tags.

**Comment by riteshjha** [comment]
I agree there should be an option to blacklist or exclude tags or css path.
